Example/SnapshotTests/BPKCalendarSnapshotTest.m

```diff
@@ -205,26 +222,26 @@ - (id _Nullable)calendar:(BPKCalendar *)calendar cellDataForDate:(BPKSimpleDate
     NSDateComponents *components = [cal components:NSCalendarUnitDay fromDate:date1 toDate:date2 options:0];
 
     if (components.day == 2 || components.day == 8 || components.day == 12 || components.day == 20) {
-        return BPKCalendarTrafficLightCellData.positive;
+        return self.isShowingPrices ? [[BPKCalendarPriceLabelCellData alloc] initWithPrice:@"£14" labelStyle:BPKCalendarPriceLabelStyle.positive] : BPKCalendarTrafficLightCellData.positive;
     }
 
     if (components.day == 4 || components.day == 10 || components.day == 24) {
-        return BPKCalendarTrafficLightCellData.negative;
+        return self.isShowingPrices ? [[BPKCalendarPriceLabelCellData alloc] initWithPrice:@"11.884.400₫" labelStyle:BPKCalendarPriceLabelStyle.negative] : BPKCalendarTrafficLightCellData.negative;
```

Comment:
In the resulting snapshot image this prices is truncated. I thought it shouldn't be 🤔

Reply:
Because the snapshot view is smaller than a real iPhone.

Reply:
Can we use a more realistic frame for this test then?

Reply:
IMO it's good to have it truncating because it means truncating is working.
We could make the screenshot bigger, and add another longer truncating case, but I can't see what it achieves?

Reply:
The snapshot tests should catch us if we accidentally break something. We've designed this version of the calendar to work correctly with the edge case of tens of million dongs i.e. "99.999.999₫" shouldn't truncate. As it stands if we break this behaviour the test wouldn't catch this regression and we wouldn't notice that we broken a valid case.

Reply:
Ok cool I'll update it then 👍